partcad/src/partcad/wrappers/wrapper_render_3mf.py

```
@@ -0,0 +1,57 @@
+# to speed up parallel rendering, and not to leverage any other benefits of sandboxing
+
+import os
+import sys
+
+import cadquery as cq
+
+sys.path.append(os.path.dirname(__file__))
+import wrapper_common
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from utils_ocp import tessellate
```

Ruff (0.8.2)

`utils_ocp.tessellate` imported but unused

Remove unused import: `utils_ocp.tessellate`

(F401)